src/lib/openapi/spec/client-features-query-schema.ts

```diff
@@ -0,0 +1,38 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const clientFeaturesQuerySchema = {
```

Comment:
Should add a few tests for the schema, just because it's part of the client API.